modules/virt-reviewing-top-consumers.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-reviewing-vm-dashboard.adoc
+
+[id="virt-reviewing-top-consumers_{context}"]
+= Reviewing top consumers
+
+In the *Administrator* perspective, you can view the {VirtProductName} dashboard where top consumers of resources are displayed.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. In the *Administrator* perspective in the {VirtProductName} web console, navigate to *Monitoring* -> *Dashboard*.
```

Comment:
I believe it has a different name in OCP 4.9. I dont have a 4.9 cluster , I need to set one up

Reply:
@sradco@redhat.com. I verified in the 4.9 cluster, it is Observe > Dashboards. I will make the change to the doc.

Reply:
Updated the content to show in the 4.9 cluster that QE provided:
Observer > Dashboards